spec/features/frontend/checkout_spec.rb

```diff
@@ -4,7 +4,7 @@
   describe "paypal payment method" do
     let(:order) { Spree::TestingSupport::OrderWalkthrough.up_to(:payment) }
     let(:paypal_payment_method) { create(:paypal_payment_method) }
-    let(:failed_response) { instance_double('response', status_code: 500) }
+    let(:failed_response) { double('response', status_code: 500) } # rubocop:disable RSpec/VerifiedDoubles
```

Comment:
[RSpec/VerifiedDoubles](https://www.rubydoc.info/gems/rubocop-rspec/1.3.1/RuboCop/Cop/RSpec/VerifiedDoubles) prefers verifying doubles over normal doubles. However, for these doubles, I chose normal doubles because the response and result objects are external dependencies of the gem. As such, the gem's tests can become brittle if the tests expect these objects to be instances of certain classes.

Reply:
Agreed, can you please move this comment in the commit description?

Reply:
@kennyadsl Okay, will move the comment to the commit description. TODO